compiler/internal/typeparams/map.go

```diff
@@ -25,40 +27,35 @@ type (
 // instance equality, objects are compared by pointer equality, and type
 // arguments with types.Identical(). To reduce access complexity, we bucket
 // entries by a combined hash of type args. This type is generally inspired by
-// typeutil.Map.
+// golang.org/x/tools/go/types/typeutil/map.go
```

Comment:
Nit/optional: Could be made a GoDoc link (assuming linking to this specific type is appropriate):

```suggestion
// [golang.org/x/tools/go/types/typeutil.Map]
```

Reply:
I forgot that's how GoDoc links are formatted, too much markdown 😄 I'll get that fixed up to a valid link and put it in GoDoc's format.